larray_editor/api.py

```diff
@@ -182,6 +186,43 @@ def view(obj=None, title='', depth=0, display_caller_info=True):
     edit(obj, title=title, readonly=True, depth=depth + 1, display_caller_info=display_caller_info)
 
 
+def _debug(stack_summary, stack_pos=None):
+    # we don't use install_except_hook/restore_except_hook so that we can restore the hook actually used when
+    # this function is called instead of the one which was used when the module was loaded.
+    orig_except_hook = sys.excepthook
+    sys.excepthook = _qt_except_hook
+
+    _app = QApplication.instance()
+    if _app is None:
+        _app = qapplication()
+        _app.setOrganizationName("LArray")
+        _app.setApplicationName("Debugger")
+        parent = None
+    else:
+        parent = _app.activeWindow()
+
+    assert isinstance(stack_summary, StackSummary)
+    dlg = MappingEditor(parent)
+    setup_ok = dlg.setup_and_check(stack_summary, stack_pos=stack_pos)
+    if setup_ok:
+        dlg.show()
+        _app.exec_()
+
+    sys.excepthook = orig_except_hook
+
+
+def debug(depth=0):
```

Comment:
add this function to the ``larray`` repository ?

Reply:
will do